include/pybind11/pybind11.h

```diff
@@ -777,6 +777,18 @@ class cpp_function : public function {
                 - delegate translation to the next translator by throwing a new type of exception. */
 
             auto last_exception = std::current_exception();
+
```

Comment:
Can you update the comment above? I.e., the "give each registered exception translator a chance to translate it to a Python exception in reverse order of registration."

Reply:
done